_documentation/troubleshooting.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Troubleshooting
+layout: doc
+level: "1"
+group: "troubleshooting"
+icon: "fas fa-bug"
+order: "70"
+---
+
+# Troubleshooting
+This page is for problems users can face.
```

Comment:
I would address users directly, something like "This page is for common problems you might face."